components/CriteriaMitra/index.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+  <Portal to="criteriamitra">
+    <div
+      :class="{
+        'modal-criteria-mitra__backdrop': true,
+        'modal-criteria-mitra__backdrop--show': show,
+        'modal-criteria-mitra__backdrop--hide': !show,
+      }"
+      @click.self="onClose('overlay')"
+    >
+      <div
+        :class="{
+          'modal-criteria-mitra__wrapper': true,
+          'modal-criteria-mitra__wrapper--show': show,
+          'modal-criteria-mitra__wrapper--hide': !show,
+        }"
+      >
+        <div class="modal-criteria-mitra__close">
+          <BaseButton variant="circle" @click="onClose">
+            <template #icon>
+              <jds-icon class="text-white" size="20px" name="times" />
+            </template>
+          </BaseButton>
+        </div>
+        <div class="modal-criteria-mitra__container">
+          <div class="modal-criteria-mitra__container-heading">
+            Kriteria Mitra Yang Dapat Mengikuti
+          </div>
+          <div class="modal-criteria-mitra__container-content">
+            <div
+              v-for="item in criteriaMitraDetails"
+              :key="item.id"
+              class="modal-criteria-mitra__container-content-main"
+            >
+              <div class="modal-criteria-mitra__container-content-main-img">
+                <img class="mx-auto" width="40px" height="40px" :src="item.image" :alt="item.textImg">
+              </div>
+              <div class="modal-criteria-mitra__container-content-main-text">
+                <h2 class="modal-criteria-mitra__container-content-main-text-title">
+                  {{ item.title }}
+                </h2>
+                <ul v-if="item.textDetails.length > 1" class="modal-criteria-mitra__container-content-main-text-info">
+                  <li
+                    v-for="(textDetail, index) in item.textDetails"
+                    :key="index"
+                    class="modal-criteria-mitra__container-content-main-text-info-message"
```

Comment:
hello @doohanas,

this class name is very long and kinda hard to understand, is it the correct BEM convention? honestly, I'm not really know much about BEM 🙏🏼

Reply:
Hello, kak @Ibwedagama,

I create this class based on BEM Convention. So, the BEM Convention is divided into three parts. 
1. Block 
Block is the top level of the component. This block should be thought of as a parent. In this case, my block is `modal-criteria-mitra`
2. Element
An element can be called a child element. In this case, my element is `container`. Then, the container has some children. So, We can add a dash or `-` to define a specific element.
3. Modifier
The modifier is to manipulate the block so that we can theme or style that particular component. We can add two dashes or `--` to define the modifier. 
The naming is too long because i have to add specific css for each element kak. 

The references: 
1. [BEM](http://getbem.com/naming/) Convention
2. BEM Convention from [css-trick.com](https://css-tricks.com/bem-101/)